src/Framework/BinaryTranslator.cs

```diff
@@ -1100,7 +1110,18 @@ public void TranslateDotNet<T>(ref T value)
 
             public void TranslateException(ref Exception value)
             {
-                TranslateDotNet<Exception>(ref value);
+                if (!ChangeWaves.AreFeaturesEnabled(ChangeWaves.Wave17_8))
+                {
+                    TranslateDotNet<Exception>(ref value);
+                    return;
+                }
+
+                if (!TranslateNullable(value))
+                {
+                    return;
+                }
+
+                BuildExceptionBase.WriteExceptionToTranslator(this, value);
```

Comment:
I was deciding between 3 options how to facilitate access to the inner state of the exceptions during de/serialization:
* Reflection (and lost easily navigability between the methods and their usage)
* Open internal access to the methods facilitationg access - so that those can be used outside of the declaring type
* Add the de/serialization utils into the base - so that they have access to privates and protected

I choosed the last one - as it was the same choice taken by the FW `Serializable` support of exceptions.

I'm not strongly opinionated about this though